tools/sign_executable/sign_executable.py

```diff
@@ -19,13 +19,13 @@
     assert sys.platform == "darwin"
 
     parser = argparse.ArgumentParser()
-    parser.add_argument("--input_dir", required=True)
-    parser.add_argument("--output_dir", required=True)
     parser.add_argument("--developer_id", required=True)
     parser.add_argument("--apple_username", required=True)
     parser.add_argument("--apple_password", required=True)
-    parser.add_argument("--temp_dir", default=os.path.join(os.path.dirname(os.path.realpath(__file__)), "tmp"))
-    parser.add_argument("--entitlements_file", default=os.path.join(os.path.dirname(os.path.realpath(__file__)), "entitlements.plist"))
+    parser.add_argument("--input_dir", default=os.path.realpath(os.path.join(os.path.dirname(__file__), "tmp", "SpearSim-Mac-Shipping-Unsigned")))
```

Comment:
The default `input_dir` in `sign_executable.py` and the default `output_dir` in `build_executable.py` do not match. They need to be matched for this script to find `SpearSim-Mac-Shipping-Unsigned` in default settings.

Reply:
I agree that the default values in `build_executable` and `sign_executable` should match. I changed the logic slightly in `sign_executable` in a way that I think is better, such that: (1) the default values match `build_executable`; and (2) the output folder matches the structure of the `-archive` step in the Unreal Build Tool, i.e., `sign_executable` doesn't modify the existing folder structure at all.